Signal/src/AppDelegate.m

```diff
@@ -190,7 +190,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
         [self application:application didReceiveRemoteNotification:remoteNotif];
     }
 
-    [self prepareScreenProtection];
```

Comment:
"Screen Lock" and "Screen protection" use the same "blocking window".  I've pulled all of that into `OWSScreenLockUI`.